src/screenControl.py

```diff
@@ -115,14 +122,27 @@ def outputBottom(self):
         borderY = maxy - 2
         # first output text since we might throw an exception during border
         usageStr = {
-            SELECT_MODE: SHORT_NAV_USAGE,
-            X_MODE: SHORT_NAV_USAGE,
+            SELECT_MODE: self.getShortNavUsageString(),
+            X_MODE: self.getShortNavUsageString(),
             COMMAND_MODE: SHORT_COMMAND_USAGE
         }[self.mode]
         borderStr = '_' * (maxx - self.getMinX() - 0)
         self.printer.addstr(borderY, self.getMinX(), borderStr)
         self.printer.addstr(borderY + 1, self.getMinX(), usageStr)
 
+    def getShortNavUsageString(self):
+        navOptions = [SHORT_NAV_OPTION_SELECTION_STR,
+                      SHORT_NAV_OPTION_NAVIGATION_STR,
+                      SHORT_NAV_OPTION_QUICK_SELECT_STR,
+                      SHORT_NAV_OPTION_COMMAND_STR]
+
+        # it does not make sense to give the user the option to "open" the selection
+        # in all-input mode
+        if not self.flags.getAllInput():
+            navOptions.insert(2, SHORT_NAV_OPTION_OPEN_STR)
```

Comment:
the insert 2 is weird here, i think could do something like

```
return ', '.join([
  SELECT_STR,
  NAV_STR,
  # comment
  None if self.flags.getAllInput() else NAV_OPEN_STR,
].filter(something()?));
```

but thats just me trying to be more functional. dont know how hard that is in practice with the filter, havent written python in a while